opwen_email_server/mailers/wikipedia.py

```diff
@@ -0,0 +1,56 @@
+from base64 import b64encode
+from datetime import datetime
+from typing import Callable
+
+from requests import get
+from wikipedia import languages
+from wikipedia import page
+from wikipedia import set_lang
+from wikipedia.exceptions import DisambiguationError
+from wikipedia.exceptions import PageError
+
+from opwen_email_server.utils.log import LogMixin
+
+WIKIPEDIA_ADDRESS = 'wikipedia@bot.lokole.ca'
+
+
+class WikipediaEmailFormatter(LogMixin):
+    def __init__(self, now: Callable[[], datetime] = datetime.utcnow):
+        self._now = now
+
+    def _get_download_link(self, url: str) -> str:
+        split_url = url.split('/')
```

Comment:
I would suggest to parse the URL using [urlparse](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) so that you can explicitly access the hostname and whatever path parts you need.

Reply:
Oh cool, perfect!